.github/workflows/build.yml

```diff
@@ -9,7 +9,15 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    uses: [actions/checkout@v3, ./.github/actions/prepare]
     steps:
-      - uses: actions/checkout@v3
-      - uses: ./.github/actions/prepare
-      - run: pnpm build
+      - id: test1
+        run: |
+          pnpm build | sed 's/^/ Build Test : '
+      - id: test2
+        run: |
+          if node ./src/index.ts ; then
+              echo $? | sed 's/^/Imports Test : success : '
+          else
+              echo $? | sed 's/^/Imports Test : failed : '
+          fi
```

Comment:
These are some very nifty shell scripts 😄 ... but they don't work on my local Macbook (macOS Monterey 12.5). Which is inconvenient enough to be a blocker given how many people use macOS.

```plaintext
$ pnpm build | sed 's/^/ Build Test : '
sed: 1: "s/^/ Build Test : ": unterminated substitute in regular expression
```

I'd suggest removing the fanciness and sticking to one command (e.g. `pnpm build`) for each test. It's _slightly_ less nice testing output but easier to run & simpler to read in this file. Then we can also remove the `id:` attribute, since they'll be more easily identifiable by the command. Thoughts?

Reply:
ok sounds good will try to fix it this weekend and send for re- review